google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -21,14 +21,13 @@
 import asyncio
 import aiohttp
 import concurrent
-import googleapiclient
-import googleapiclient.discovery
 import google.auth
 from google.auth.credentials import Credentials
 import google.auth.transport.requests
 import json
-import OpenSSL
```

Comment:
Are there updates needed to setup.py requirements?

Reply:
Definitely, I'm currently taking care of that in #21